engines/sci/sound/decoders/sol.cpp

```diff
@@ -140,7 +251,8 @@ SOLStream<STEREO, S16BIT, OLDDPCM8>::SOLStream(Common::SeekableReadStream *strea
 	_stream(stream, disposeAfterUse),
 	_sampleRate(sampleRate),
 	// SSCI aligns the size of SOL data to 32 bits
-	_rawDataSize(rawDataSize & ~3) {
+	_rawDataSize(rawDataSize & ~3),
+	_popfixDPCM8({ConfMan.getBool("popfix_enabled"), 0, 0}) {
```

Comment:
Two things:
1. I would rather call `ConfMan` from somewhere else and store the result, but SOLStream's structure doesn't make it easy to do that easy, so that's my problem to deal with later; I don't think you should worry about it.
2. How to say this... can you make this more boring? =)
I don't even know how to read this C syntax; can you break it up so that you're explicitly setting fields and not creating a scope within a thing... I'm going to gloss over my limitations and just say "our existing code doesn't use this convention" =)

Reply:
Ah no worries - I did check before I used it; that inline structure init syntax is actually used elsewhere the codebase, but no problems at all to change it! =)

I'll add a constructor to the internal struct to achieve the same result, which would make it far more obvious I think. Assigning the other two values there would put them next to the actual members rather than being "magic zeros" too.